shared/constants/chat.js

```diff
@@ -845,6 +849,10 @@ const getParticipantsWithFullNames = createSelector(
   }
 )
 
+const getTLF = createSelector([getParticipants], participants => {
```

Comment:
just a note i think we shouldn't have shared selectors ever. you can't know who's invalidating your cache so i think its just a really bad idea